docs/_importers/behance.md

```diff
@@ -1,21 +1,7 @@
 ---
-layout: docs
 title: Behance
 importer: true
 prev_section: usage
 link_source:  behance
 next_section: blogger
-permalink: /docs/behance/
 ---
-
-To import your posts from your [Behance](http://behance.com), generate an API token for your user account and run:
-
-{% highlight bash %}
-$ ruby -r rubygems -e 'require "jekyll-import";
-    JekyllImport::Importers::Behance.run({
-      "user"      => "my_username",
-      "api_token" => "my_api_token"
-    })'
-{% endhighlight %}
```

Comment:
The plugin you wrote is super cool and the final output is amazing! I'm kind of sad that a reader of this Jekyll site source wouldn't actually get to see much of the content that they'd view in the output. It simultaneously shows the power of Jekyll ("you can inject all sorts of content at build time!") and the drawback of that power ("a lot of the key content on the site isn't in the source files"). I wonder if it would make sense for either (1) us to generate the content into the site source which we push up to the repo or (2) the plugin to inject a variable into the Liquid template so our page can at least have a reference like `{{ auto_generated_usage_guide }}`. Some food for thought. Right now it's very magical but I'm debating if maybe it separates the source and final output too much?

Reply:
For hosted metadata, we could generate another data file for CLI information using a Rake task just like we have one for the list of importer-dependencies.

But that is less exciting (for me, atleast) than having content generated automatically at runtime, especially because contributors / maintainers have to ensure the data file is updated as needed.
For example, say we add a new option to an importer or we make edits to the option-description, then
  - the data file has to be explicitly updated to ensure the change gets documented.
  - Maintainers would have to explicitly check if the pull request has the data file updated to match the proposed change before merging the PR.

Regarding the Liquid object `autogenerated...`, that too *is possible*, but I am not yet clear on the suggested use-case.

Reply:
Good points. Let's leave it how you have it now.